src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -374,3 +374,45 @@ export function validateBooleanGroup(
     errors.addError(atLeastOne);
   }
 }
+
+/* Military history validations */
+export const validateAge = (
+  errors,
+  dateString,
+  _formData,
+  _schema,
+  _uiSchema,
+  _currentIndex,
+  appStateData,
+) => {
+  if (moment(dateString).isBefore(moment(appStateData.dob).add(13, 'years'))) {
```

Comment:
If you wanted to, I think you could also do this with [date-fns](https://date-fns.org/v2.19.0/docs/isBefore), possibly (completely untested) something like: 

```
import isBefore from 'date-fns/isBefore'
import parseISO from 'date-fns/parseISO'
import add from 'date-fns/add'

if (isBefore(parseISO(dateString)), add(parseISO(appStateData.dob), {years: 13}))) {...}
```

Reply:
Hehe yeah, I didn't want to mess with the actual validation function just yet. I did use `date-fns` for the unit tests. It was weird when I initially did this 13 year-old test, it passed locally, but failed in CI, so I had to subtract one day